src/monitor/restart/conditions/ForkedCluster.ts

```diff
@@ -0,0 +1,154 @@
+import IRestartCondition, {
+  ShouldRestartInfo,
+} from '@interfaces/restart-conditions/IRestartCondition';
+import IAllowanceListService from '@interfaces/services/allowance-list/IAllowanceListService';
+import IGlobalNetworkService from '@interfaces/services/global-network/IGlobalNetworkService';
+import ILoggerService from '@interfaces/services/logger/ILoggerService';
+import IMetagraphService from '@interfaces/services/metagraph/IMetagraphService';
+import ISeedlistService from '@interfaces/services/seedlist/ISeedlistService';
+import ISshService from '@interfaces/services/ssh/ISshService';
+import { Layers } from '@shared/constants';
+import { Config, MonitoringConfiguration } from 'src/MonitoringConfiguration';
+
+import { FullLayer } from '../groups';
+
```

Comment:
I think a comment that describes the scenario this condition aims to detect would be a good idea here.